core/internal/tensorboard/tensorboard.go

```diff
@@ -320,19 +321,20 @@ func (tb *TBHandler) convertToRunHistory(
 }
 
 func (tb *TBHandler) sendHistoryRequest(request *service.PartialHistoryRequest) {
-	tb.outChan <- &service.Record{
-		RecordType: &service.Record_Request{
-			Request: &service.Request{
-				RequestType: &service.Request_PartialHistory{
-					PartialHistory: request,
+	tb.extraWork.AddRecord(
```

Comment:
it says it's not covered by tests, how come?

Reply:
I believe this line was not covered before.